Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -65,6 +65,8 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed an issue where selection state of UI Toolkit editor state would not be preserved when associated with a new serialized copy of the asset.
 - Fixed an issue where any exceptions throw from within UI Toolkit event queue would only log the error message and not the full exception stack trace, making debugging more difficult.
 - Fixed an issue where UI Toolkit Input Action Editor wouldn't provide a correct modification state when coming back from domain reload.
+- Fixed duplication of project wide input actions when loading/unloading scenes.
```

Comment:
Is this line about duplication a merge error. Looks like its unrelated to the fix.

Reply:
I have cleaned this up when merging latest develop into this branch, there was a bad merge conflict resolution introducing a duplicate.